### PR TITLE
[codestyle] Fixed casing in function/method names

### PR DESCRIPTION
**Pull request by ghost**
This is a larger one and could use some thorough looking over and testing.
Also please check that I haven't changed files from external packages.

PS: The coding Standards doc doesn't specifically state that "traditionally uppercase acronyms (such as XML, HTML)" should also be camelcased in function/method names.
http://joomla.github.io/coding-standards/?coding-standards/chapters/php.md
But it does seem to make the most sense.
If it is also a rule that covers function/method names, it would be good to state that more clearly in the docs.

### CODE REVIEW

File: administrator/components/com_admin/script.php [hunk 1]
```diff
@@ -49,7 +49,7 @@ protected function updateDatabase()
 
 		if (strpos($db->name, 'mysql') !== false)
 		{
-			$this->updateDatabaseMySQL();
+			$this->updateDatabaseMySql();
```
Comment (Bakual) [inline administrator/components/com_admin/script.php]: I would treat `MySQL` as one word and thus use `Mysql` here. Probably sneaked in using search-replace?

File: libraries/vendor/joomla/uri/src/AbstractUri.php [hunk 2]
```diff
@@ -290,7 +290,7 @@ public function getFragment()
 	 *
 	 * @since   1.0
 	 */
-	public function isSSL()
+	public function isSsl()
```
Comment (Bakual) [inline libraries/vendor/joomla/uri/src/AbstractUri.php]: Needs to be changed upstream in the framework

File: libraries/vendor/joomla/registry/src/Format/Xml.php [hunk 3]
```diff
@@ -42,7 +42,7 @@ public function objectToString($object, $options = array())
 		// Iterate over the object members.
 		$this->getXmlChildren($root, $object, $nodeName);
 
-		return $root->asXML();
+		return $root->asXml();
```
Comment (Bakual) [inline libraries/vendor/joomla/registry/src/Format/Xml.php]: Needs to be changed upstream in the framework

File: libraries/vendor/joomla/registry/src/Format/Ini.php [hunk 4]
```diff
@@ -91,12 +91,12 @@ public function objectToString($object, $options = array())
 						foreach ($v as $array_key => $item)
 						{
 							$array_key = ($assoc) ? $array_key : '';
-							$local[] = $k . '[' . $array_key . ']=' . $this->getValueAsINI($item);
+							$local[] = $k . '[' . $array_key . ']=' . $this->getValueAsIni($item);
```
Comment (Bakual) [inline libraries/vendor/joomla/registry/src/Format/Ini.php]: Needs to be changed upstream in the framework

File: libraries/vendor/joomla/uri/src/UriInterface.php [hunk 5]
```diff
@@ -147,5 +147,5 @@ public function getFragment();
 	 *
 	 * @since   1.0
 	 */
-	public function isSSL();
+	public function isSsl();
```
Comment (Bakual) [inline libraries/vendor/joomla/uri/src/UriInterface.php]: Needs to be changed upstream in the framework

File: tests/unit/suites/libraries/joomla/mail/JMailTest.php [hunk 6]
```diff
@@ -143,34 +143,34 @@ public function testAddRecipient()
 	/**
 	 * Tests the addCC method.
 	 *
-	 * @covers  JMail::addCC
+	 * @covers  addCC::addCc
```
Comment (Bakual) [inline tests/unit/suites/libraries/joomla/mail/JMailTest.php]: This looks wrong :smile:

File: tests/unit/suites/libraries/joomla/mail/JMailTest.php [hunk 7]
```diff
@@ -143,34 +143,34 @@ public function testAddRecipient()
...
-		$this->object->addCC($recipient, $name);
+		$this->object->addCc($recipient, $name);
 		$this->assertThat($expected, $this->equalTo(TestReflection::getValue($this->object, 'cc')));
 	}
 
 	/**
 	 * Tests the addBCC method.
 	 *
-	 * @covers  JMail::addBCC
+	 * @covers  addBCC::addBcc
```
Comment (Bakual) [inline tests/unit/suites/libraries/joomla/mail/JMailTest.php]: same here

File: administrator/components/com_admin/script.php [hunk 8]
```diff
@@ -60,7 +60,7 @@ protected function updateDatabase()
 	 *
 	 * @return void
 	 */
-	protected function updateDatabaseMySQL()
+	protected function updateDatabaseMySql()
```
Comment (Bakual) [inline administrator/components/com_admin/script.php]: Same here of course.

**Comment by Bakual** [comment]
Looks good besides the things I commented inline.

File: tests/unit/suites/libraries/joomla/mail/JMailTest.php (hunk 7 above)
Comment (nonumber) [inline tests/unit/suites/libraries/joomla/mail/JMailTest.php]: fixed

File: tests/unit/suites/libraries/joomla/mail/JMailTest.php (hunk 6 above)
Comment (nonumber) [inline tests/unit/suites/libraries/joomla/mail/JMailTest.php]: fixed

File: administrator/components/com_admin/script.php (hunk 1 above)
Comment (nonumber) [inline administrator/components/com_admin/script.php]: fixed

**Comment by nonumber** [comment]
Thanks for the feedback.
Fixed the issues.
And will create a PR towards joomla-framework too.

**Comment by Bakual** [comment]
> And will create a PR towards joomla-framework too.

Can you revert the changes done in the vendor folder files?
Otherwise I can do before merging myself.

**Comment by nonumber** [comment]
done. :)

**Comment by Bakual** [comment]
Perfect, thanks!

**Comment by nonumber** [comment]
For the PR to joomla-framework, see:
https://github.com/joomla/joomla-framework/pull/377

**Pull request closed without merging**